src/main/java/com/maybe/maybe/entity/Invoice.java

```diff
@@ -37,6 +38,10 @@
     @JsonBackReference
     private Set<InvoiceItem> invoiceItems;
 
+    public Invoice() {
+        this.setInvoiceItems(new HashSet<>());
```

Comment:
I think it is looks like better:
private Set<InvoiceItem> invoiceItems = new HashSet<>();